cmake/Scripts/WriteVerSHA.cmake

```diff
@@ -22,6 +22,18 @@ if (NOT ${has_sha} EQUAL 0)
   set (sha1 "")
 endif ()
 
+
+exec_program(
+  date
+  ARGS +"%d-%m-%Y at %H:%m:%S hrs"
+  OUTPUT_VARIABLE build_timestamp
+  RETURN_VARIABLE has_timestamp)
+
+if (NOT ${has_timestamp} EQUAL 0)
+  set (build_timestamp "")
+endif()
```

Comment:
I think you can even do this directly in CMake:
```cmake
string (TIMESTAMP build_timestamp "%Y-%m-%d at %H:%M:%S hrs")
````
That way you don't need to depend on having an executable `date` in the search path.

Reply:
OK I'll update - just using CMake is obviously better.